static/js/sefaria/sefaria.js

```diff
@@ -1964,6 +1964,9 @@ _media: {},
         store: this._TopicsByPool
     });
   },
+    getLangSpecificTopicPoolName: function(poolName, lang){
+      return `${poolName}_${lang}`
```

Comment:
This function should not take lang as a parameter and instead should use Sefaria.interfaceLang internally. This is reasonable since this function is located in sefaria.js and pool name is supposed to be tied to the interface language.